js_modules/dagit/packages/core/typings.d.ts

```diff
@@ -28,3 +28,13 @@ declare module 'worker-loader!*' {
 }
 
 declare module 'chartjs-adapter-date-fns';
+
+declare namespace Intl {
+  type Key = 'calendar' | 'collation' | 'currency' | 'numberingSystem' | 'timeZone' | 'unit';
+
+  function supportedValuesOf(input: Key): string[];
+
+  interface Locale extends LocaleOptions {
+    timeZones: string[];
```

Comment:
I wonder if we could get this into our env by changing something in our base CRA fork? Doesn't seem super important though and this is likely easier, just imagine having to update it a bit as we continue to use more of Intl

Reply:
My bet is that it will be included in future TS versions.